src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveService.kt

```diff
@@ -111,9 +111,10 @@ class OppgaveService constructor(private val oppgaveRestClient: OppgaveRestClien
 
     fun opprettOppgave(request: OpprettOppgaveRequest): Long {
         val oppgave = Oppgave(
-                identer = if (request.ident != null) listOf(request.ident!!) else null,
+                identer = if (erFolkeregisteridentEllerNpid(request.ident)) listOf(request.ident!!) else null,
                 aktoerId = if (request.ident?.gruppe == IdentGruppe.AKTOERID) request.ident!!.ident else null,
                 orgnr = if (request.ident?.gruppe == IdentGruppe.ORGNR) request.ident!!.ident else null,
+                samhandlernr = if (request.ident?.gruppe == IdentGruppe.SAMHANDLERNR) request.ident!!.ident else null,
```

Comment:
Ser egentlig ikke denne i PostOppgaveRequest

Reply:
Den mangler i swagger doc'en, men ser ut som den blir lagret for det:
[https://github.com/navikt/oppgave/blob/master/src/main/java/no/nav/oppgave/oppgave/json/PostOppgaveRequestJson.java#L182](https://github.com/navikt/oppgave/blob/master/src/main/java/no/nav/oppgave/oppgave/json/PostOppgaveRequestJson.java#L182)